catalog/utilities/media_props_gen/helpers/db.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import re
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any
+
+from common.constants import MediaType
+
+
+LOCAL_POSTGRES_FOLDER = Path(__file__).parents[4] / "docker" / "upstream_db"
+SQL_PATH = {
+    "image": LOCAL_POSTGRES_FOLDER / "0003_openledger_image_schema.sql",
+    "audio": LOCAL_POSTGRES_FOLDER / "0006_openledger_audio_schema.sql",
+}
+SQL_TYPES = [
+    "integer",
+    "boolean",
+    "uuid",
+    "double precision",
+    "jsonb",
+    "timestamp with time zone",
+    "character varying",
```

Comment:
Interestingly, the output from the API table shows this as `varchar(<length>)` and not `character varying`.

Reply:
TIL: In Postgres, you can use both interchangeably (https://www.postgresql.org/docs/current/datatype-character.html#DATATYPE-CHARACTER).
I guess we could replace `character varying` with `varchar(<length>)` in the catalog docs and in the local catalog SQL files for consistency.

Reply:
TIL as well. `varchar` is technically shorter so fits better in a narrow table 🤷 but that's not justification to use one over the other.

Reply:
Consistency there would be nice though! Would one of you mind making a follow-up issue for it?